javascript/javascript2/week3/task_2.js

```diff
@@ -0,0 +1,44 @@
+//step-1
+const array = [myName, myAge, myAddress];
+array.forEach((element) => {
+  element();
+});
+
+function myName(name) {
+  let name = "John";
+  console.log("My name is" + " " + name);
+}
+function myAge(age) {
+  const age = "20 yrs";
+  console.log("My age is" + "  " + age);
+}
+function myAddress(address) {
+  const address = "street 10,Copenhagen";
+  console.log("My address is" + " " + address);
+}
```

Comment:
You don' need to pass name, age, address as parameters, since you are declaring them inside functions.
For me it gives an error in the console:
![image](https://user-images.githubusercontent.com/31403464/132060634-db7233ad-7528-4d4c-ad63-39ba6ba22ef4.png)


Also you can use template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals for strings in console.log().
Like: console.log(\`My name is ${name}\`);

Reply:
done!